lib/ebay_api.rb

```diff
@@ -1,73 +1,80 @@
 require 'multi_xml'
 require 'net/http'
 
-module EbayAPI
-  class EbayApiError < StandardError
-    attr_accessor :request, :response
+module OmniAuth
+  module Strategies
+    class EbayAPI
```

Comment:
`OmniAuth::Strategies` is not appropriate namespace, actually. Because this is not strategy itself and OmniAuth should not try to load it. It is better to keep it somewhere in gem namespace. Something like `OmniAuth::Ebay::EbayAPI` (scary, I know!)